getting_started/install_openshift.adoc

```diff
@@ -0,0 +1,229 @@
+== Overview
+
+This guide introduces you to the basic concepts of {product-title}, and helps
+you install a basic application. This guide is
+not suitable for deploying or installing a production environment of OpenShift.
+
+[[install-prerequisites]]
+=== Prerequisites
+
+To install {product-title}, you will need:
+
+* At least two physical or virtual RHEL 7+ machines, with fully qualified domain
+names (either real world or within a network) and password-less ssh access to
+each other. This guide, uses master.openshift.example.com and
+node.openshift.example.com. These machines must be able to ping each other
+using these domain names.
+* A valid Red Hat subscription.
+* Wildcard DNS resolution that resolves your domain to the IP of the node. So,
+an entry like the following in your DNS server:
+
+----
+master.openshift.example.com. 300 IN  A <IP of the master>
+node.openshift.example.com. 300 IN A <IP of the node>
+*.apps.openshift.example.com. 300 IN  A <IP of the node>
+----
+
+[NOTE]
+.Why the *apps* in your domain name for the wildcard entry?
+====
+When using {product-title} to deploy applications, an internal router needs to proxy
+incoming requests to the corresponding application pod. By using *apps* as part
+of the application domains, the application traffic is accurately marked to the
+right pod.
+
+You can use anything other than *apps*. For example,
+
+----
+*.cloudapps.openshift.example.com. 300 IN  A <IP of the node>
+----
+====
+
+Once you have acquired these items, use these steps to set up a two-machine
+{product-title} install.
+
+[[attach-subscription]]
+=== Attach {product-title} Subscription
+
+. On the target machines (both master and node), as root, use
+`subscription-manager` to register the systems with Red Hat:
++
+----
+$ subscription-manager register
+----
+
+. List the available subscriptions:
++
+----
+$ subscription-manager list --available
+----
+
+. Find the pool ID that provides {product-title} Subscription and attach it:
++
+----
+$ subscription-manager attach --pool=<pool_id>
+----
+
+. Replace the string `<pool_id>` with the pool ID of the pool that provides
+{product-title} 3. The pool ID is a long alphanumeric string.
+
+These RHEL systems are now authorized to install {product-title}. Now you need
+to tell the systems from where to get {product-title}.
+
+[[set-up-repositories]]
+=== Set Up Repositories
+
+On both master and node, use `subscription-manager` to enable the repositories
+that are necessary in order to install {product-title} (you may have already enabled
+the first two repositories in this example):
+
+----
+$ subscription-manager repos --enable="rhel-7-server-rpms" --enable="rhel-7-server-extras-rpms" --enable="rhel-7-server-ose-3.6-rpms" --enable="rhel-7-fast-datapath-rpms"
+----
+
+[NOTE]
+====
+Instead of `rhel-7-server-ose-3.6-rpms`, use `rhel-7-server-ose-3.5-rpms` for
+OpenShift Container Platform 3.5 and `rhel-7-server-ose-3.4-rpms` for OpenShift
+Container Platform 3.4
```

Comment:
This could be a bit stronger. 
"If you are using OpenShift Container Platform 3.5 use `rhel-7-server-ose-3.5-rpms`.
If you are using OpenShift Container Platform 3.4 use `rhel-7-server-ose-3.4-rpms`.